content/posts/newsletter-february-2025.md

```diff
@@ -0,0 +1,33 @@
+title: Read the Docs Newsletter - February 2025
+date: February 11, 2025
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community
+authors: Eric Holscher
+status: published
+image: /images/feb-2025.jpg
+image_credit: Photo by <a href="https://unsplash.com/@sunshinevillage?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Banff Sunshine Village</a> on <a href="https://unsplash.com/photos/2-person-in-yellow-jacket-and-blue-helmet-riding-ski-blades-on-snow-covered-mountain-during-UoBE_wJ-suk?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+## News and updates
+
+* We migrated our official documentation to ``docs.readthedocs.com``, with the primary user documentation at https://docs.readthedocs.com/platform/stable/.
+* We are working to improve the process of changing the slug of a Version. This will make it easier to change the URL of a version without creating a new version. This is currently being tested in production and will be available soon.
+* We [added support for changing the flyout position in Addons](https://docs.readthedocs.com/platform/stable/flyout-menu.html#position), which allows you to control where the flyout appears on your documentation site.
+* Our [Visual diff](https://docs.readthedocs.io/en/stable/visual-diff.html) feature is getting closer to being widely released. This includes a user-facing UX for seeing what files changed in a PR build, and a way to see the visual diff.
+* We [documented Addons customization patterns](https://docs.readthedocs.com/platform/stable/addons.html#addons-data-and-customization) to help you understand how to customize Addons for your project.
+
+## Upcoming changes
```

Comment:
Maybe worth to note we are working on flyout position inline, https://github.com/readthedocs/addons/pull/500, and see if we can get some feedback from a theme author.

Reply:
That seems like something for social media or direct pings on GH, more than newsletter.